salt/modules/win_iis.py

```diff
@@ -1300,6 +1348,10 @@ def set_container_setting(name, container, settings):
     failed_settings = dict()
 
     for setting in settings:
+        # map identity type from numeric to string for comparing
+        if (container == 'AppPools' and setting == 'processModel.identityType'):
+            settings[setting] = identity_type_map(settings[setting],'string')
+
```

Comment:
If you go with the dictionary approach above, then you wouldn't need the `'string'` parameter.